nltk/tokenize/punkt.py

```diff
@@ -570,8 +570,8 @@ def _tokenize_words(self, plaintext):
                 yield self._Token(tok, parastart=parastart, linestart=True)
                 parastart = False
 
-                for t in line_toks:
-                    yield self._Token(t)
+                for token in line_toks:
```

Comment:
Prefer to be consistent with abbreviations, e.g. `for tok in line_toks` or `for token in line_tokens`